docs/contributing-to-translations.md

```diff
@@ -108,6 +106,4 @@ and tell you what's wrong.
 
 ## Add a Language
 
-To add a new language, add an entry to `frontend.dicts/languages`. Then add a
-new locale keyword to `frontend.dicts/dicts` and to
-`frontend.modules.shortcut.dicts/dicts` and start translating as described above.
+To add a new language, add an entry to `frontend.dicts.core/languages`. Then add a new locale keyword to `frontend.dicts.core/dicts`.
```

Comment:
Adding a language is a little more involved than this e.g. the user also has to define two vars for their file and wire them up to the right vars

Reply:
I believe that now it is more explainable the process of adding a new language